contrib/devtools/test-security-check.py

```diff
@@ -21,7 +21,7 @@ def write_testcode(filename):
 
 def call_security_check(cc, source, executable, options):
     subprocess.check_call([cc,source,'-o',executable] + options)
-    p = subprocess.Popen(['./security-check.py',executable], stdout=subprocess.PIPE, stderr=subprocess.PIPE, stdin=subprocess.PIPE, universal_newlines=True)
+    p = subprocess.Popen(['../contrib/devtools/security-check.py',executable], stdout=subprocess.PIPE, stderr=subprocess.PIPE, stdin=subprocess.PIPE, universal_newlines=True)
```

Comment:
This will break when the distdir is more than one folder deep. I'd prefer if the script was linked like everything else (test_runner.py, etc)